idowuadeleke/BANKA
# #165656221 revamp data format

#### What does this PR do?
- revamp data response format

### Description of Task to be completed?
- ensure all data responses are returned as an array
### How should this be manually tested?
- clone the repo `git clone https://github.com/idowuadeleke/BANKA.git`

- on your terminal run `cd BANKA`

- run `git checkout ch-165656221-revamp-data-format`

- run `npm install`

- run `npm run start`

- make a request to `POST localhost:3000/api/v1/auth/signup`  and see data returned as an array on success

### Any background context you want to provide?
NA
### What are the relevant pivotal tracker stories?
[#165656221](https://www.pivotaltracker.com/story/show/165656221)

### Screenshots (if appropriate)ensure user input are well validated
N/A
